src/UnityExtension/Assets/Editor/GitHub.Unity/UI/ProjectWindowInterface.cs

```diff
@@ -169,11 +171,14 @@ private static void OnLocksUpdate(IEnumerable<GitLock> update)
                 var g = AssetDatabase.AssetPathToGUID(assetPath);
                 guidsLocks.Add(g);
             }
+
+            EditorApplication.RepaintProjectWindow();
         }
 
         private static void RunStatusUpdateOnMainThread(GitStatus update)
         {
-            EntryPoint.ApplicationManager.TaskManager.ScheduleUI(new ActionTask(EntryPoint.ApplicationManager.TaskManager.Token, _ => OnStatusUpdate(update)));
+            new ActionTask(EntryPoint.ApplicationManager.TaskManager.Token, _ => OnStatusUpdate(update))
+                .ScheduleUI(EntryPoint.ApplicationManager.TaskManager);
```

Comment:
IF you're changing this to a `new ActionTask`, then it should be `new ActionTask(...) { Affinity = TaskAffinity.UI }.Start()`, otherwise there's no point in making this change.

Reply:
Should I update this section as well

https://github.com/github-for-unity/Unity/blob/9c52781d810deb9cb91190438703d1300138e121/src/UnityExtension/Assets/Editor/GitHub.Unity/UI/ProjectWindowInterface.cs#L151-L155